### DISCUSSION TITLE
How to give parameter of a command a description?

### DISCUSSION THREAD

**Discussion by coolbrett** [Pycord-Development/pycord#1861] [Q&A] [answered]
I have this code:

```python
@bot.command(name="standings", description="Current Standings for Fantasy League", guild_ids=[guild_id])
async def standings(interaction: discord.Interaction, year: int = None):
```

and here is how it looks in discord:
![image](https://user-images.githubusercontent.com/38200353/211131563-04412b05-fa06-44e9-a966-4e2174c72928.png)

How do I add a description to the optional parameter `year`?

**Comment by JustaSqu1d** [comment, marked as answer]
Using the example, we can use the `option` decorator to add a name and a description to the option like so:
```python
@bot.slash_command()
@discord.option(
    name = "year"
    description = "What year will it be?"
)
async def command(ctx, year: int):
    pass
```

I would like to add that this discussion category should not be used for help, rather, please join our [Discord server](https://pycord.dev/discord) for help instead. If you require further assistance please do the same.

**Reply by coolbrett** [reply]
Thank you for the quick reply! and I'll make sure to post questions in the discord from now on
